modules/serverless-go-template.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-go-functions.adoc
+
+[id="serverless-go-template_{context}"]
+= Golang function template structure
+
+When you create a Golang function using the `kn func` CLI, the project directory looks like a typical Go project, with the exception of an additional `func.yaml` configuration file.
+
+Golang functions have few restrictions. The only requirement is that your project must be defined in a `function` module, and must export the function `Handle()`.
+
+Both `http` and `event` trigger functions have the same template structure:
+
+.Template structure
+[source,terminal]
+----
+fn
+├── README.md
+├── func.yaml <1>
+├── go.mod <2>
+├── go.sum
+├── handle.go
+└── handle_test.go
+----
+<1> The `func.yaml` configuration file is used to determine the image name and registry.
+<2> You can add any required dependencies to the `go.mod` file, which can include additional local Golang files. When the project is built for deployment, these dependencies will be included in the resulting runtime container image.
++
+.Example of adding dependencies
```

Comment:
Just checking, is this supposed to be indented/included with callout 2? And I wasn't sure exactly what you were supposed to do with this, is it a command to run in the terminal, or is it a line that is supposed to be added to the go.mod file? 

If the former, needs a $ prompt. If the latter, maybe make that clearer in the heading text?

Reply:
Added the prompt, thanks